.builder/actions/crt-ci-test.py

```diff
@@ -27,29 +27,89 @@ def _write_secret_to_temp_file(self, env, secret_name):
 
         return filename
 
+    def _build_and_run_eventstream_echo_server(self, env):
+        java_sdk_dir = None
+        proc = None
+
+        try:
+            env.shell.exec(["mvn", "--version"], check=True)
+
+            # maven is installed, so this is a configuration we can start an event stream echo server
+            java_sdk_dir = env.shell.mktemp()
+
+            env.shell.exec(["git", "clone", "https://github.com/aws/aws-iot-device-sdk-java-v2"], working_dir=java_sdk_dir, check=True)
+
+            sdk_dir = os.path.join(java_sdk_dir, "aws-iot-device-sdk-java-v2", "sdk")
+            env.shell.pushd(sdk_dir)
+
+            try:
+                env.shell.exec(["mvn", "compile"], check=True)
+                env.shell.exec(["mvn", "test", "-DskipTests=true"], check=True)
```

Comment:
This will probably do the trick, and makes a bit more sense than "test, but don't test"
```suggestion
                # the echo server is a test class
                env.shell.exec(["mvn", "test-compile"], check=True)
```